runtimes/alibi-detect/tests/conftest.py

```diff
@@ -0,0 +1,21 @@
+import pytest
+import os
+from mlserver.types import InferenceRequest
+
+
+TESTS_PATH = os.path.dirname(__file__)
+TESTDATA_PATH = os.path.join(TESTS_PATH, "testdata")
+
+
+def pytest_collection_modifyitems(items):
+    """
+    Add pytest.mark.asyncio marker to every test.
+    """
+    for item in items:
+        item.add_marker("asyncio")
```

Comment:
If you rebase from `master` (this PR in particular: #507 ), this bit shouldn't be required anymore.

Reply:
Fixed now in 8cdec90